applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -309,8 +321,10 @@ static bool app_event_handler(const struct app_event_header *aeh)
 		process_dvfs_states(DVFS_STATE_CONFIG_CHANNEL, true);
```

Comment:
The Kconfig options that enable/disable handing of application events need to be updated. Otherwise you would call `process_dvfs_states` even if user explicitly disabled DVFS frequency updates for given state.